samples/data-bid-ask/bidask.py

```diff
@@ -87,7 +87,9 @@ def runstrategy():
 
     cerebro = bt.Cerebro()  # Create a cerebro
 
-    data = BidAskCSV(dataname=args.data, dtformat=args.dtformat)
```

Comment:
This is simply generic, not meant to be Ticks per se.

Reply:
Again this will be your output
```
   1: 02/03/2010 23:59:59.999989 - Bid 0.5346 - 0.5347 Ask
   2: 02/03/2010 23:59:59.999989 - Bid 0.5343 - 0.5347 Ask
   3: 02/03/2010 23:59:59.999989 - Bid 0.5543 - 0.5545 Ask
   4: 02/03/2010 23:59:59.999989 - Bid 0.5342 - 0.5344 Ask
   5: 02/03/2010 23:59:59.999989 - Bid 0.5245 - 0.5464 Ask
   6: 02/03/2010 23:59:59.999989 - Bid 0.5460 - 0.5470 Ask
   7: 02/03/2010 23:59:59.999989 - Bid 0.5824 - 0.5826 Ask
   8: 02/03/2010 23:59:59.999989 - Bid 0.5371 - 0.5374 Ask
   9: 02/03/2010 23:59:59.999989 - Bid 0.5793 - 0.5794 Ask
  10: 02/03/2010 23:59:59.999989 - Bid 0.5684 - 0.5688 Ask
```

The time is not dispalying correctly. What was the purpose of this sample then, just to show Bid and Ask price?

Reply:
Again this will be your output
```
   1: 02/03/2010 23:59:59.999989 - Bid 0.5346 - 0.5347 Ask
   2: 02/03/2010 23:59:59.999989 - Bid 0.5343 - 0.5347 Ask
   3: 02/03/2010 23:59:59.999989 - Bid 0.5543 - 0.5545 Ask
   4: 02/03/2010 23:59:59.999989 - Bid 0.5342 - 0.5344 Ask
   5: 02/03/2010 23:59:59.999989 - Bid 0.5245 - 0.5464 Ask
   6: 02/03/2010 23:59:59.999989 - Bid 0.5460 - 0.5470 Ask
   7: 02/03/2010 23:59:59.999989 - Bid 0.5824 - 0.5826 Ask
   8: 02/03/2010 23:59:59.999989 - Bid 0.5371 - 0.5374 Ask
   9: 02/03/2010 23:59:59.999989 - Bid 0.5793 - 0.5794 Ask
  10: 02/03/2010 23:59:59.999989 - Bid 0.5684 - 0.5688 Ask
```

The time is not dispalying correctly. What was the purpose of this sample then, just to show Bid and Ask price?